occupancy-quickstart/src/auth.cs

```diff
@@ -44,21 +44,27 @@ private static async Task<bool> TryRequestWithAccessToken(Uri baseAddress, strin
         private static string ReadAccessTokenFromFile(string filename)
             => System.IO.File.Exists(filename) ? System.IO.File.ReadAllText(filename) : null;
 
-        private static async Task<string> GetNewToken(
-            ILogger logger,
-            AppSettings appSettings)
-        {
-            var authContext = new Microsoft.IdentityModel.Clients.ActiveDirectory.AuthenticationContext(appSettings.Authority);
-            return (await GetResultsUsingDeviceCode(authContext, appSettings)).AccessToken;
-        }
+        private static async Task<string> GetNewToken(AppSettings appSettings)
+            => await GetResultsUsingMsal(appSettings);
 
-        // This prompts the user to open a browser and input a unique key to authenticate their app
-        // This allows dotnet core apps to authorize an application through user credentials without displaying UI.
-        private static async Task<AuthenticationResult> GetResultsUsingDeviceCode(AuthenticationContext authContext, AppSettings appSettings)
+        // MSAL.NET configuration. Review the product documentation for more information about MSAL.NET authentication options.
+        // https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/
+        private static async Task<string> GetResultsUsingMsal(AppSettings appSettings)
```

Comment:
nit: I would either return AuthenticationResult like it use to here (and change the caller to do the .AccessToken like it use to.  Or change the name to GetTokenUsingMsal or GetNewToken and get rid of the GetNewToken above.

Reply:
Thanks! Took the last option and removed the original `GetNewToken()` and renamed `GetTokenUsingMsal()`.